build.sbt

```diff
@@ -37,7 +37,7 @@ lazy val root = (project in file(".")).enablePlugins(
       "com.gu.memsub.Subscription.ProductRatePlanId"
   ))
 
-scalaVersion := "2.12.10"
+scalaVersion := "2.12.14"
```

Comment:
oppostunistic bump, but we could do with going to 2.13 soon, as membership-common is already bumped.